index.html

```diff
@@ -0,0 +1,555 @@
+                        </div>
+                        <div class="flex-item">
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/LEGO-Super-Mario-796x417.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>LEGO Super Mario combines 2 staples of childhood</p>
+                                <small>Rachael Kaser <i class="fa fa-circle"></i>15 hours ago</small>
+                            </div>
+                        </div>
+                        <div class="flex-item">
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2015/12/Sundar-Pichai-Google-for-India-1200x600.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>No clue how to handle Corona Virus as a start-up founder? Here's what we are doing</p>
+                                <small>Patrick De Laive <i class="fa fa-circle"></i>15 hours ago</small>
+                            </div>
+                        </div>
+                        <div class="flex-item">
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/08/Discord-Go-Live-516x270.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>White House asks Silicon Valley for AI solutions to Corona Virus</p>
+                                <small>Thomas Macaulay <i class="fa fa-circle"></i>14 hours ago</small>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+
+                <div class="funding">
+                    <div class="fund-header">
+                        <h1>Latest funding rounds on Index.co</h1>
+                        <p>Add Info | Get Updates by mail | See more</p>
+                    </div>
+                    <div class="co-details">
+                        <div class="fund-item">
+                            <img src="https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1177352942887653376%2FkF_JJ65n.jpg&signature=5b89445a9415b121f9ff891b53f029cb" alt="Albert-logo">
+                            <h4>Shopback</h4>
+                            <div class="fund">
+                                <p> $30M </p>
+                                <small>SERIES B</small>
+                            </div>
+                            <div class="fund2">
+                                <p><i class="tag icon"></i> E-commerce, +2</p>
+                                <p><i class="tag icon"></i> Rakuten, +5 </p>
+                            </div>
+                            <div class="fund3">
+                                <small> SG's Cash back App Shopback closes $75M Series B led by Tamasek On Dealstreetasia </small>
+                            </div>
```

Comment:
- [ ] Check the original width and padding of the elements in this section, yours look less wide and have a lot of padding.